### ISSUE TITLE
RPN Predictions always only cover top half of the image?

### ISSUE THREAD

**Issue by bgkw** [matterport/Mask_RCNN#1089]
I run the code in inspect_model to evaluate the model. I found the RPN training part is working well. But in RPN predictions, there are 6000 pre_nms_anchors. Display shows all those anchors boxes are ALWAYS on top half of the image. It seems the grid of anchors is never covering the whole image. What’s wrong with my config? Thanks. 

I use COCO weights and no augmentation. And the ground truth boxes should be distributed randomly in the training dataset.  

    IMAGE_MIN_DIM = 768
    IMAGE_MAX_DIM = 768
    
    RPN_ANCHOR_SCALES = (8, 16, 32, 64,128),
    RPN_ANCHOR_RATIOS = [0.5, 1, 2]    
    # How many anchors per image to use for RPN training
    RPN_TRAIN_ANCHORS_PER_IMAGE = 128    
    RPN_NMS_THRESHOLD = 0.9
    
    # Number of ROIs per image to feed to classifier/mask heads
    TRAIN_ROIS_PER_IMAGE = 256
    ROI_POSITIVE_RATIO = 0.2
    
    # ROIs kept after non-maximum supression (training and inference)
    POST_NMS_ROIS_TRAINING = 1024
    POST_NMS_ROIS_INFERENCE = 1024

**Comment by Robinchacko** [comment]
@bgkw Could you please let me know what all things that you have tried to overcome this issue?

**Comment by bgkw** [comment]
I don't remember the details. But it's not a bug of Mask_RCNN, it was something wrong with my Mask RCNN config.

    On Thursday, March 28, 2019, 6:23:36 p.m. EDT, Robinchacko <notifications@github.com> wrote:  
 
 
@bgkw Could you please let me know what all things that you have tried to overcome this issue?
